website/src/documentation/sdks/nexmark.md

```diff
@@ -171,18 +173,24 @@ Number of events generators:
 
     --numEventGenerators=4
 
-Run query N:
+Queries can be run by their name or by their number (number is still there for backward compatibility, only the queries 0 to 12 have a number)
+
+Run query **N**:
 
     --query=N
 
+Run query called **PASSTROUGH**:
```

Comment:
Typo: missing `H` and also below

Reply:
Wait actually still missing the H

Reply:
oups, this conversation was left resolved so I missed this comment. I'll fix in another PR then